content/articles/changing-plans.md

```diff
@@ -36,8 +36,9 @@ You can cancel these plan changes in your account settings:
 ## Changing your subscription plan
 
 1.  Log in to DNSimple with your user credentials.
-2.  [Find the account](https://dnsimple.com/user) from the account list. Click on the account to enter the account page.
-3.  Click the **Change plan** link under your plan section.
+2.  Select the relevant account from the account switcher in the top-right corner of the screen.
```

Comment:
Can we do all `1.`?

Reply:
definitely, we've been working to remove the "log in" step in all articles since it's unnecessary